app/src/main/res/layout/activity_manage_card.xml

```diff
@@ -0,0 +1,373 @@
+            android:layout_marginEnd="5dp"
+            app:cardBackgroundColor="@color/background"
+            app:cardElevation="10dp"
+            android:outlineSpotShadowColor="@color/background">
+
+            <androidx.constraintlayout.widget.ConstraintLayout
+                android:layout_width="match_parent"
+                android:layout_height="match_parent">
+
+                <androidx.cardview.widget.CardView
+                    android:layout_width="match_parent"
+                    android:layout_height="match_parent"
+                    app:layout_constraintTop_toTopOf="parent"
+                    app:layout_constraintBottom_toBottomOf="parent"
+                    app:layout_constraintEnd_toEndOf="parent"
+                    app:layout_constraintStart_toStartOf="parent"
+                    app:cardBackgroundColor="@color/black"
+                    android:layout_margin="3dp"
+                    app:cardCornerRadius="17dp"/>
+
+            </androidx.constraintlayout.widget.ConstraintLayout>
+
+        </androidx.cardview.widget.CardView>
+
+
+    </androidx.constraintlayout.widget.ConstraintLayout>
+
+    <androidx.cardview.widget.CardView
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        app:layout_constraintTop_toBottomOf="@+id/cardLayout"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        android:layout_marginHorizontal="16dp"
+        app:cardCornerRadius="13dp"
+        app:cardBackgroundColor="@color/white"
+        app:cardElevation="60dp"
+        android:outlineSpotShadowColor="@color/background"
+        android:layout_marginTop="15dp">
+
+        <androidx.constraintlayout.widget.ConstraintLayout
+            android:layout_width="match_parent"
+            android:layout_height="match_parent">
+
+            <androidx.constraintlayout.widget.ConstraintLayout
+                android:id="@+id/freezeCardLayout"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                app:layout_constraintTop_toTopOf="parent"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintEnd_toEndOf="parent">
+
+                <androidx.cardview.widget.CardView
+                    android:id="@+id/pinkCard"
+                    android:layout_width="36dp"
+                    android:layout_height="36dp"
+                    android:layout_marginVertical="21dp"
+                    app:layout_constraintTop_toTopOf="parent"
+                    app:layout_constraintBottom_toBottomOf="parent"
+                    app:layout_constraintStart_toStartOf="parent"
+                    android:layout_marginStart="16dp"
+                    app:cardCornerRadius="8dp"
+                    app:cardBackgroundColor="@color/pink" />
+
+                <androidx.constraintlayout.widget.ConstraintLayout
```

Comment:
You shouldn't use nested `ConstraintLayout's` you can only use one `ConstraintLayout` parent for all views.

Reply:
I used it in that way to be more regular writing, it's sort of wrapper and behaves like just one element and items within that wrapper behaves same. For example that layout has a margin and have an impact of all elements. I thought this is better solution.

Reply:
> [ConstraintLayout](https://developer.android.com/reference/androidx/constraintlayout/widget/ConstraintLayout) allows you to create large and complex layouts with a flat view hierarchy (no nested view groups).

Look at the first sentence of this doc :) They created `ConstraintLayout` at the first place to prevent using nested view groups.

https://developer.android.com/develop/ui/views/layout/constraint-layout